app/controllers/application_controller.rb

```diff
@@ -12,7 +12,12 @@ def include_google_analytics?
   end
 
   def set_visitor_id
-    if cookies[:visitor_id].present?
+    return if current_intake&.visitor_id.present? && current_intake&.visitor_id == cookies[:visitor_id]
```

Comment:
added an early return because this happens as a before action on like everything?

Reply:
seems like a win from a code formatting perspective. I'm not sure the performance implications are very large because there aren't any database calls in the logic that this skips.

Reply:
hm true! still not great with performance stuff haha. i think i'll remove the line so the whole thing is a little less confusing